src/scripts/components/DropdownMultiSelectAdvanced.jsx

```diff
@@ -0,0 +1,233 @@
+      <div className="dropdown__menu__container">
+        <div className="search__group">
+          <UtilityInlineGrid className="u-flex u-flex-justify-between u-m-t-small u-text-small">
+            {this.renderClearButton()}
+            <div>
+              <Button
+                size="small"
+                type="link"
+                onClick={this.handleToggle}
+                title="Back"
+              >Back
+              </Button>
+            </div>
+          </UtilityInlineGrid>
+        </div>
+        {this.props.selectedItemsIds.length > 0 ? (
+          <Scrollbars className="resource-group__scroll" autoHeight autoHeightMax={UtilitySystem.config.resourceSizes.large}>
+            <ResourceGroup interfaceMode="checkbox">
+              {this.props.selectedItemsIds.map(this.renderSelectedItemsList)}
+            </ResourceGroup>
+          </Scrollbars>
+        ) :
+          'No items Added'
+        }
+      </div>
+    </Dropdown>
+  );
+
+  render() {
+    const { itemSearchLoading, dropdownLabel, selectedItemsIds, filterName } = this.props;
+    const itemsIds = [...this.props.itemsIds];
+    const searchTitle = `Search ${filterName}`;
+    let returnValue = '';
+    returnValue = (
+      this.state.isViewAllItems ? (
+        <Dropdown wide autoFocusInput={false} label={dropdownLabel} onClick={this.clearSearch} type="outline-primary" disableScroll>
+          <div className="dropdown__menu__container">
+            <div className="search__group">
+              {selectedItemsIds.length > 0 ? (
+                <UtilityInlineGrid className="u-flex u-flex-justify-between u-m-t-small u-text-small">
+                  {this.renderClearButton()}
+                  {this.renderViewSelected()}
+                </UtilityInlineGrid>
+              ) : (
+                <ResourceRight>
+                  {this.renderViewSelected()}
+                </ResourceRight>)
+              }
+              <Input
+                placeholder={searchTitle}
+                className="search__input"
+                onChange={this.handleSearch}
+                initialValue={this.state.searchText}
+                addon="left"
+                type="text"
+                name="preloadedMembers"
+                autoComplete="off"
+              >
+                <Icon icon="search" />
+              </Input>
+            </div>
+          </div>
+          <div className="dropdown__menu__container">
+            {itemsIds.length > 0 ? (
+              <Scrollbars className="resource-group__scroll" autoHeight autoHeightMax={UtilitySystem.config.resourceSizes.large}>
+                <ResourceGroup interfaceMode="checkbox">
+                  {itemsIds.map(this.renderList)}
+                </ResourceGroup>
+              </Scrollbars>
+            ) :
+              this.renderSearchHelp(itemsIds, itemSearchLoading)
+              }
```

Comment:
I don't think this bracket lines up with the opening bracket.

Reply:
@alexgsummer Ok I will change it before merge